### ISSUE TITLE
otm-dach-contours file is missing

### ISSUE THREAD

**Issue by stefan1983** [der-stefan/OpenTopoMap#320]
On the download page the otm-dach-contours file is missing.

Thanks

**Comment by der-stefan** [comment]
That is true. I accidentally deleted that file some time ago and did not recreate it yet. As this file does not change, I asked the community if someone who downloaded it can provide it to me. That was not the case.

**Comment by cristian5th** [comment]
Hello.
Any chance to recreate the missing file? Do you have some instructions on how to do it from the OSM original data? I can help if you tell me how.
Thank you for all your work. Regards.
